src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -71,15 +98,51 @@ public void close() throws Exception {
     // Mostly for JUnit tests
     motor.close();
     motor2.close();
-    
   }
 
-  @Override
-  public void periodic() {
-    intakeMode.setString((isConeMode) ? "Cone Mode" : "Cube Mode");
-    // This method will be called once per scheduler run
+  public double getConeDist() {
+    return timeOfFlightSensor.getRawDistance0().classDistance;
   }
 
+  public double getCubeDist() {
+    return timeOfFlightSensor.getRawDistance1().classDistance;
+  }
+
+  public void changeFlightState() {
+    double coneDistance = getConeDist();
+    double cubeDistance = getCubeDist();
+
+    boolean coneSensorUp = coneDistance != -1;
```

Comment:
There is an isSensor0Connected method, let's use that instead of figuring out on our own.

That was if the logic changes, we don't have to change it in two places.

Also, we should only get the coneDistance if the coneSensor is up.

Reply:
Changed to only check for distance if sensors are up. If not, values are kept at -1